grpc-server/worker_workflow.go

```diff
@@ -8,28 +8,13 @@ import (
 
 	"github.com/packethost/pkg/log"
 	"github.com/tinkerbell/tink/db"
-	pb "github.com/tinkerbell/tink/protos/workflow"
+	"github.com/tinkerbell/tink/protos/workflow"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 )
 
-var workflowData = make(map[string]int)
```

Comment:
This map was accessed and set in a non-concurrent-safe method, but was essentially unused for any real logic. I just removed it

Reply:
confirmed